code/modules/games/tarot.dm

```diff
@@ -168,3 +169,221 @@
 		P.back_icon = "card_off_[suit]"
 		cards += P
 
+/// Lyodii tarot deck.
+/obj/item/deck/tarot/lyodii
+	name = "lyodii fatesayer deck"
+	icon = 'icons/obj/item/playing_cards.dmi'
+	icon_state = "lyodii_deck"
+	desc = "A traditionally-made deck of Fatesayer cards, used by the people of the Lyod to tell one's fate."
+	desc_extended = "These 'cards' are actually rectangular pieces of bone, engraved with different religious imagery. They are then painted with soot or blood-ink. Usually made by tribal shamans in \
+	agonizingly difficult work, these are hand-crafted by each tribe - thus some imagery can deviate from one another. A deck consists of 36 pieces, divided as follows: Spirits, Beasts, Winds and Paths \
+	each having eight cards in their stack and Bones, with four cards. Traditionally a Bone card is drawn, then one each of the rest. The Bone cards put the other four cards into perspective so one's fate \
+	can be determined. More modern iterations of these Fatesayer decks are also made by the few permanent settlements in the Lyod, to sell them off. They are not using real bone and the imagery is usually more \
+	refined and detailed, since they are machine-made."
+	drop_sound = 'sound/items/drop/bone_drop.ogg'
+	pickup_sound = 'sound/items/drop/bone_drop.ogg'
+	hand_type = /obj/item/hand/tarot/lyodii
+
+/obj/item/deck/tarot/lyodii/AltClick(var/mob/user)
+	playsound(src.loc, 'sound/items/cards/bone_shuffle.ogg', 100, 1, -4)
+	user.visible_message("\The [user] rearranges \the [src].")
```

Comment:
1) add `if(use_check_and_message(src)) return` at the start

2) SPAN_NOTICE("[user] rearranges \the [src].")